client/src/hoc/Auth.tsx

```diff
@@ -23,30 +22,40 @@ const Auth = (Component: any, option: boolean) => () => {
 
       try {
         if (token) {
```

Comment:
```suggestion
        if (token) {
} else if (!option) {
}
```
처럼 바꿀수 있겠네요.
아래 success 도 비슷하고요.

Reply:
넵 감사합니다! 답변이 늦어서 죄송합니다 ㅠㅜ

제가 이렇게 if else를 쓰지 않고 if만 쓴 이유는 예전에 리뷰를 받았을때, else의 사용을 지양하라고 하셨어서.. 이렇게 했던 것인데, if else 사용은 문제가 없을까요?